File: 1 - two sum/twoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
    for (const[index, number] of nums.entries()) {
        for(const[otherIndex, otherNumber] of nums.entries()){
            if (index === otherIndex) continue;
            if (number + otherNumber === target) return [index, otherIndex];
        }
    }
};

const nums = [-1,2,3,1,2,7,20];
const target = 9;
console.log(twoSum(nums, target));

File: 1 - two sum/two_sum.py
class Solution:
    def twoSum(self, nums, target):
        for index, num in enumerate(nums):
            for i, n in enumerate(nums):
                if index == i:
                    continue
                if num + n == target:
                    return [index, i]

nums = [3,3]
target = 6
solution = Solution()
print(solution.twoSum(nums, target))